lib/props/props.test.js

```diff
@@ -0,0 +1,14 @@
+import props from '.'
+
+test('it doesn\'t extract props if element of keys is not an array', () => {
+  expect(() => props('a', { a: 1, b: 2 })).toThrowError('Path should be an array')
+  expect(() => props(['a', 'b'], { a: 1, b: 2 })).toThrowError('Path should be an array')
```

Comment:
Here is the path is Array, why should it throw error? It should return `[1, 2]` instead

Reply:
fixed